x-pack/plugins/observability_solution/synthetics/server/synthetics_service/project_monitor/normalizers/http_monitor.ts

```diff
@@ -50,9 +52,11 @@ export const getNormalizeHTTPFields = ({
   errors.push(...commonErrors);
 
   /* Check if monitor has multiple urls */
-  const urls = getOptionalListField(monitor.urls);
+  const urls = getUrlsField(monitor.urls);
   if (urls.length !== 1) {
     errors.push(getInvalidUrlsOrHostsError(monitor, 'urls', version));
+  } else if (isValidURL(urls[0]) === false) {
```

Comment:
could we do this instead?
```suggestion
  } else if (!isValidURL(urls[0])) {
```